packages/rich-text/README.md

```diff
@@ -47,7 +46,7 @@ Create an HTML string from a Rich Text value. If a `multilineTag` is provided, t
 apply( {
 	value: Object,
 	current: Element,
-	?multilineTag: string
+	?multilineTag: string,
```

Comment:
Do we have stub documentation of the format itself? Would be great to.

Reply:
I added something to `create` in the `readme.md`.